PopPUNK/utils.py

```diff
@@ -591,3 +600,27 @@ def decisionBoundary(intercept, gradient):
     x = intercept[0] + intercept[1] * gradient
     y = intercept[1] + intercept[0] / gradient
     return(x, y)
+
+def check_and_set_gpu(use_gpu, gpu_lib):
```

Comment:
Would a quit option be helpful, which causes this to quit rather than continue? I think that's done in one use of this